### DISCUSSION TITLE
planetscale and postman

### DISCUSSION THREAD

**Discussion by darbok** [planetscale/discussion#598] [Q&A] [answered]
so i can get a postman get request to return the path and name of my database, but I can't seem to get it to Post to one of the database tables that I know works cause I've used console to insert into it. Not sure what I'm missing.

**Comment by orware** [comment, marked as answer]
Hello @darbok,

I believe I received your request when it was sent in via our support site (https://support.planetscale.com/) and hopefully the answers I've provided there helped to address the questions you had!.

For others coming here, a quick summary to share is that the Public API (https://api-docs.planetscale.com/reference) can be used to perform certain PlanetScale actions such as creating branches, deploy requests, etc. but that API is not for directly interacting with the database and tables and issuing queries. 

For that you would either connect to your database [using the MySQL protocol](https://planetscale.com/docs/tutorials/connect-any-application) or use our [serverless driver for JavaScript](https://planetscale.com/docs/tutorials/planetscale-serverless-driver) to run and issue queries. Should you want to, users can build out their own API layer for interacting with your database and tables.

**Comment by darbok** [comment]
I'm looking for it, but can't find it. What confuses me is I saw a youtube video of this lady adding data to a table using postman.
     On Thursday, January 25, 2024 at 06:53:27 PM EST, Omar Ramos ***@***.***> wrote:  
 
 


Hello @darbok,

I believe I received your request when it was sent in via our support site (https://support.planetscale.com/) and hopefully the answers I've provided there helped to address the questions you had!.

For others coming here, a quick summary to share is that the Public API (https://api-docs.planetscale.com/reference) can be used to perform certain PlanetScale actions such as creating branches, deploy requests, etc. but that API is not for directly interacting with the database and tables and issuing queries.

For that you would either connect to your database using the MySQL protocol or use our serverless driver for JavaScript to run and issue queries. Should you want to, users can build out their own API layer for interacting with your database and tables.
